server/migrations/20200527155244-add-notif-email-to-user-model.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    await queryInterface.addColumn(
+      'Users',
+      'emailNotificationInterval',
+      {
+        type: DataTypes.ENUM,
+        values: ['daily', 'weekly', 'monthly', 'never'],
+        allowNull: false,
+        defaultValue: 'never', // TODO: do we want to strategically set this?
```

Comment:
never is good, I'd remove the TODO unless @raykyri thinks otherwise.

Reply:
I removed it. We can always attend to setting people's email via a migration (those who haven't updated will now be set to weekly, or similar)